supported-configurations.json

```
@@ -0,0 +1,913 @@
+      "version": ["A"]
+    },
+    "DD_TRACE_SEQUEL_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SEQUEL_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SEQUEL_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SHORYUKEN_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SHORYUKEN_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SHORYUKEN_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SIDEKIQ_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SIDEKIQ_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SIDEKIQ_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SINATRA_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SINATRA_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SINATRA_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SNEAKERS_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SNEAKERS_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SNEAKERS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_STARTUP_LOGS": {
+      "version": ["A"]
+    },
+    "DD_TRACE_STRIPE_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_STRIPE_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_STRIPE_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SUCKER_PUNCH_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SUCKER_PUNCH_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_SUCKER_PUNCH_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TEST_MODE_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TRILOGY_ANALYTICS_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TRILOGY_ANALYTICS_SAMPLE_RATE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TRILOGY_ENABLED": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TRILOGY_PEER_SERVICE": {
+      "version": ["A"]
+    },
+    "DD_TRACE_TRILOGY_SERVICE_NAME": {
+      "version": ["A"]
+    },
+    "DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH": {
+      "version": ["A"]
+    },
+    "DD_VERSION": {
+      "version": ["A"]
+    },
+    "OTEL_TRACES_SAMPLER_ARG": {
+      "version": ["A"]
+    }
+  },
+  "aliases": {
+    "DD_DISABLE_DATADOG_RAILS": ["DISABLE_DATADOG_RAILS"],
+    "DD_PROFILING_GVL_ENABLED": ["DD_PROFILING_PREVIEW_GVL_ENABLED"],
+    "DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED": ["DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED"],
+    "DD_SERVICE": ["OTEL_SERVICE_NAME"],
+    "DD_TAGS": ["OTEL_RESOURCE_ATTRIBUTES"],
+    "DD_TRACE_DEBUG": ["OTEL_LOG_LEVEL"],
+    "DD_TRACE_ENABLED": ["OTEL_TRACES_EXPORTER"],
+    "DD_TRACE_PROPAGATION_STYLE": ["OTEL_PROPAGATORS"],
+    "DD_TRACE_SAMPLE_RATE": ["OTEL_TRACES_SAMPLER"]
+  },
+  "deprecations": {
+    "DD_PROFILING_PREVIEW_GVL_ENABLED": "(This doesn't need a deprecation message since there's an alias for it)"
```

Would this not log `This does not need a deprecation message ...`?

I think it should be:

```suggestion
    "DD_PROFILING_PREVIEW_GVL_ENABLED": "DD_PROFILING_GVL_ENABLED"
```

That way it would log the alias accordingly. Looking into this seems important to me before landing.

---

Makes sense, thank you for pointing that out. I guess it's fine with just testing for that.